frontend/src/pages/Admin/Components/JoinUs/Card/card.module.scss

```diff
@@ -0,0 +1,56 @@
+.card-item {
+    text-align: center;
+    font-size: 1.5rem;
+    border-radius: 1em;
+    height: auto;
+    width: 15em;
+    margin: 1em;
+    display: inline-block;
+    background-position: left center;
+    transition: all 0.5s ease-in;
+    background-color: #016795;
+    box-shadow: 0.5em 0.5em 0.5em rgb(54, 53, 53);
+  }
+  
```

Comment:
Please format this file. Each class should be separated by a single blank line and the code should be formatted